cluster/vagrant/provision-master.sh

```diff
@@ -32,7 +32,7 @@ systemctl restart network
 
 function release_not_found() {
   echo "It looks as if you don't have a compiled version of Kubernetes.  If you" >&2
-  echo "are running from a clone of the git repo, please run ./build/release.sh." >&2
+  echo "are running from a clone of the git repo, please run 'KUBE_RELEASE_RUN_TESTS=n ./build/release.sh.'" >&2
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  tapiohush@gmail.com
    Domain eikaas.no has exceeded the max emails per hour (32/30 (106%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext5.iad.github.net ([192.30.252.196]:34249 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by cpanel30.proisp.no with esmtps (TLSv1.2:DHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.85)
	(envelope-from <noreply@github.com>)
	id 1ZEhaw-0020fU-2a
	for robin@eikaas.no; Mon, 13 Jul 2015 19:32:52 +0200
Date: Mon, 13 Jul 2015 10:32:44 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1436808764;
	bh=d3zuv60wgoIAocB5Ej24u+gdPkOUVSwJ+31O+r3Rp6k=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=BalmR05LbILcStkiT3BVPL4++Z+tKFwVDqKFkZGhrE//3kkmoq+gZJA4mLA94UpL0
	 SOxocNCrD4wCiQpzDElTmuIb5E8+yjVvWuKgVg2uX7MtyKCxKIj57NI5OjtoY1876D
	 Ay6wn3kTL3IlRMj7TDhlSzyaYTsUd4b+riLqU/oM=
From: Jason Riddle <notifications@github.com>
Reply-To: GoogleCloudPlatform/kubernetes <reply@reply.github.com>
To: GoogleCloudPlatform/kubernetes <kubernetes@noreply.github.com>
Message-ID: <GoogleCloudPlatform/kubernetes/pull/11120/r34488418@github.com>
In-Reply-To: <GoogleCloudPlatform/kubernetes/pull/11120@github.com>
References: <GoogleCloudPlatform/kubernetes/pull/11120@github.com>
Subject: Re: [kubernetes] Add KUBE_RELEASE_RUN_TESTS=n to suggestion (#11120)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_55a3f63cdff0d_65c43f8b403f52b8112428c";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: jason-riddle
X-GitHub-Recipient: eikaas
List-ID: GoogleCloudPlatform/kubernetes
 <kubernetes.GoogleCloudPlatform.github.com>
List-Archive: https://github.com/GoogleCloudPlatform/kubernetes
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+005f5f6703181a2ec96374a75b36b7c583703ab819c2ba9592cf0000000111bbb83c92a170ce020e4062@reply.github.com>,
 <https://github.com/notifications/unsubscribe/AF9fZzZmZvarD-rj2o0jH0Dlyz_kqwAQks5oc-28gaJpZM4FWzDM>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: robin@eikaas.no
X-Spam-Status: No, score=-0.9
X-Spam-Score: -8
X-Spam-Bar: /
X-Ham-Report: Spam detection software, running on the system "cpanel30.proisp.no",
 has NOT identified this incoming email as spam.  The original
 message has been attached to this so you can view it or label
 similar future email.  If you have any questions, see
 root\@localhost for details.
 
 Content preview:  > @@ -32,7 +32,7 @@ systemctl restart network > > function
    release_not_found() { > echo "It looks as if you don't have a compiled version
    of Kubernetes. If you" >&2 > - echo "are running from a clone of the git
   repo, please run ./build/release.sh." >&2 > + echo "are running from a clone
    of the git repo, please run 'KUBE_RELEASE_RUN_TESTS=n ./build/release.sh.'"
    >&2 [...] 
 
 Content analysis details:   (-0.9 points, 4.0 required)
 
  pts rule name              description
 ---- ---------------------- --------------------------------------------------
  0.0 URIBL_BLOCKED          ADMINISTRATOR NOTICE: The query to URIBL was blocked.
                             See
                             http://wiki.apache.org/spamassassin/DnsBlocklists#dnsbl-block
                              for more information.
                             [URIs: github.com]
 -0.0 SPF_PASS               SPF: sender matches SPF record
 -1.9 BAYES_00               BODY: Bayes spam probability is 0 to 1%
                             [score: 0.0000]
  0.0 HTML_MESSAGE           BODY: HTML included in message
  1.1 HTML_IMAGE_ONLY_16     BODY: HTML: images with 1200-1600 bytes of words
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from author's
                             domain
 -0.1 DKIM_VALID             Message has at least one valid DKIM or DK signature
  0.1 DKIM_SIGNED            Message has a DKIM or DK signature, not necessarily valid
X-Spam-Flag: NO


----==_mimepart_55a3f63cdff0d_65c43f8b403f52b8112428c
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -32,7 +32,7 @@ systemctl restart network
>  
>  function release_not_found() {
>    echo "It looks as if you don't have a compiled version of Kubernetes.  If you" >&2
> -  echo "are running from a clone of the git repo, please run ./build/release.sh." >&2
> +  echo "are running from a clone of the git repo, please run 'KUBE_RELEASE_RUN_TESTS=n ./build/release.sh.'" >&2

Works for me

---
Reply to this email directly or view it on GitHub:
https://github.com/GoogleCloudPlatform/kubernetes/pull/11120/files#r34488418
----==_mimepart_55a3f63cdff0d_65c43f8b403f52b8112428c
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11120#discussion_r34488418">cluster/vagrant/provision-master.sh</a>:</p>
<pre style='color:#555'>&gt; @@ -32,7 +32,7 @@ systemctl restart network
&gt;  
&gt;  function release_not_found() {
&gt;    echo &quot;It looks as if you don&#39;t have a compiled version of Kubernetes.  If you&quot; &gt;&amp;2
&gt; -  echo &quot;are running from a clone of the git repo, please run ./build/release.sh.&quot; &gt;&amp;2
&gt; +  echo &quot;are running from a clone of the git repo, please run &#39;KUBE_RELEASE_RUN_TESTS=n ./build/release.sh.&#39;&quot; &gt;&amp;2
</pre>
<p>Works for me</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11120/files#r34488418">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/AF9fZ4EmgxreClGrG1ljML8to405vl7mks5oc-28gaJpZM4FWzDM.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/GoogleCloudPlatform/kubernetes/pull/11120/files#r34488418"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_55a3f63cdff0d_65c43f8b403f52b8112428c--